rskj-core/src/main/java/org/ethereum/rpc/dto/TransactionResultDTO.java

```diff
@@ -79,4 +77,117 @@ public TransactionResultDTO(Block b, Integer index, Transaction tx) {
             s = TypeConverter.toQuantityJsonHex(signature.getS());
         }
     }
+
+    public String getHash() {
+        return hash;
+    }
+
+    public void setHash(String hash) {
+        this.hash = hash;
+    }
+
+    public String getNonce() {
+        return nonce;
+    }
+
+    public void setNonce(String nonce) {
+        this.nonce = nonce;
+    }
+
+    public String getBlockHash() {
+        return blockHash;
+    }
+
+    public void setBlockHash(String blockHash) {
+        this.blockHash = blockHash;
+    }
+
+    public String getBlockNumber() {
+        return blockNumber;
+    }
+
+    public void setBlockNumber(String blockNumber) {
+        this.blockNumber = blockNumber;
+    }
+
+    public String getTransactionIndex() {
+        return transactionIndex;
+    }
+
+    public void setTransactionIndex(String transactionIndex) {
+        this.transactionIndex = transactionIndex;
+    }
+
+    public String getFrom() {
+        return from;
+    }
+
+    public void setFrom(String from) {
+        this.from = from;
+    }
+
+    public String getTo() {
+        return to;
+    }
+
+    public void setTo(String to) {
+        this.to = to;
+    }
+
+    public String getGas() {
+        return gas;
+    }
+
+    public void setGas(String gas) {
+        this.gas = gas;
+    }
+
+    public String getGasPrice() {
+        return gasPrice;
+    }
+
+    public void setGasPrice(String gasPrice) {
+        this.gasPrice = gasPrice;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    public String getInput() {
+        return input;
+    }
+
+    public void setInput(String input) {
+        this.input = input;
+    }
+
+    public String getV() {
+        return v;
+    }
+
+    public void setV(String v) {
+        this.v = v;
+    }
+
+    public String getR() {
+        return r;
+    }
+
+    public void setR(String r) {
+        this.r = r;
+    }
+
+    public String getS() {
+        return s;
+    }
+
+    public void setS(String s) {
+        this.s = s;
+    }
```

Comment:
It seems that these "setter methods" are not being used anywhere, I would remove them since it helps with understanding how we use this DTO in the rest of the code. What do you think?

Reply:
Also "getter methods" for BlockNumber, Gas, and GasPrice seems unused, I don't know what to think about this... maybe we have to refactor this and delete these fields?

Reply:
removed the setters, the getters are necessary for testing purposes. Also since it's a dto, it should only be used as dumb presentational object, any other use would break that pattern.
all the involved fields are necessary to match the api spec, there's no need to remove any field

Reply:
and I'm not completely sure but maybe jackson it's using the getters also